core/pom.xml

```diff
@@ -60,6 +60,33 @@
       </build>
     </profile>
   </profiles>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin</artifactId>
+        <version>3.5.0</version>
+
+		<executions>
+			<execution>
+				<goals>
+					<goal>bnd-process</goal>
+				</goals>
+				</execution>
+		</executions>
+		</plugin>
+
+		<plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-jar-plugin</artifactId>
+			<configuration>
+				<archive>
+					<manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
```

Comment:
This should be the default, no?

Reply:
In my tests the maven jar plugin ignores it otherwise :-(